packages/amplify-provider-awscloudformation/src/graphql-transformer/utils.ts

```diff
@@ -39,6 +44,7 @@ export const getIdentityPoolId = async (ctx: $TSContext): Promise<string | undef
 
 export const getAdminRoles = async (ctx: $TSContext, apiResourceName: string): Promise<Array<string>> => {
   const currentEnv = ctx.amplify.getEnvInfo().envName;
+  const backEndDir = ctx.amplify.pathManager.getBackendDirPath();
```

Comment:
nit: use pathManager
```suggestion
  const backEndDir = pathManager.getBackendDirPath();
```